### ISSUE TITLE
nmcli method4 auto

### ISSUE THREAD

**Issue by goegol** [ansible/ansible-lint#4070]
##### Summary

Ansible-lint is complaining with a warn that the parameter ofr method4 in the plugin nmcli is wrong.

```
args[module]: Unsupported parameters for (basic.py) module: method4. Supported parameters include: ageingtime, arp_interval, arp_ip_target, autoconnect, conn_name, dhcp_client_id, dns4, dns4_search, dns6, dns6_search, downdelay, egress, flags, forwarddelay, gw4, gw6, hairpin, hellotime, ifname, ingress, ip4, ip6, ip_tunnel_dev, ip_tunnel_local, ip_tunnel_remote, mac, master, maxage, miimon, mode, mtu, path_cost, primary, priority, slavepriority, state, stp, type, updelay, vlandev, vlanid, vxlan_id, vxlan_local, vxlan_remote. (warning)
```

According to the docs its correct: https://docs.ansible.com/ansible/latest/collections/community/general/nmcli_module.html#parameter-method4

method4 supports the following options:
 - "auto"
 - "link-local"
 - "manual"
 - "shared"
 - "disabled"

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint 24.2.1 using ansible-core:2.16.4 ansible-compat:4.1.11 ruamel-yaml:0.18.6 ruamel-yaml-clib:0.2.8
```

- ansible installation method: OS package
- ansible-lint installation method: pip

##### STEPS TO REPRODUCE

```console (paste below)
- name: Add a second ip
  community.general.nmcli:
    conn_name: System eth0
    state: present
    type: ethernet
    ifname: eth0
    ip4: "192.168.1.3"
    method4: "auto"
```

##### Desired Behavior

Needs to accept auto as valid option for method4

##### Actual Behavior

Ansible-lint is complaining that auto is not a supported option for method4:
```
args[module]: Unsupported parameters for (basic.py) module: method4. Supported parameters include: ageingtime, arp_interval, arp_ip_target, autoconnect, conn_name, dhcp_client_id, dns4, dns4_search, dns6, dns6_search, downdelay, egress, flags, forwarddelay, gw4, gw6, hairpin, hellotime, ifname, ingress, ip4, ip6, ip_tunnel_dev, ip_tunnel_local, ip_tunnel_remote, mac, master, maxage, miimon, mode, mtu, path_cost, primary, priority, slavepriority, state, stp, type, updelay, vlandev, vlanid, vxlan_id, vxlan_local, vxlan_remote. (warning)
```

**Comment by audgirka** [comment]
@goegol Thanks for this issue. However, I'm unable to reproduce this with the above mentioned version and also with the latest version of the tool.

![Screenshot 2024-04-23 at 14 14 43](https://github.com/ansible/ansible-lint/assets/11896437/bf8c76d4-59d4-4b42-9509-a0edf6d1bf26)

Closing it for now.

Thanks
